test/test_sparse_csr.py

```diff
@@ -1062,10 +1063,15 @@ def _test(t, x, y):
                     expected = torch.addmm(t, x.to_dense(), y.to_dense())
                     self.assertEqual(res, expected)
 
+                def _test_mm(x, y):
                     res = torch.mm(x, y)
                     expected = torch.mm(x.to_dense(), y.to_dense())
                     self.assertEqual(res, expected)
```

Comment:
This is interesting. Either `mm(sparse_compressed, sparse_compressed)` returns strided tensor or one can use `assertEqual` on operands having different layouts.

Reply:
Thanks, @pmeier, for the info! A solution is to redefine `assertEqual` locally to use `exact_layout=True` when sensible.